listeners/syntax.py

```diff
@@ -64,15 +66,25 @@ def _handle_codeerrors(self, data):
                 region_that_would_be_looked_up = self.view.word(reg.begin())
                 if region_that_would_be_looked_up.begin() != reg.begin() or region_that_would_be_looked_up.end() != reg.end():
                     reg = sublime.Region(point, point+1)
-                self.underlines.append(reg)
+                # self.underlines.append(reg)
+                if i["LogLevel"] == "Warning" :
+                    self.warninglines.append(reg)
+                if i["LogLevel"] == "Error" :
+                    self.errlines.append(reg)
                 key = "%s,%s" % (reg.a, reg.b)
                 oops_map[key] = i["Text"].strip()
                 self.outputpanel.run_command('append', {'characters': i["LogLevel"] + " : " + i["Text"].strip() + " - (" + str(i["Line"]) + ", " + str(i["Column"]) + ")\n"})
-            if len(self.underlines) > 0:
-                print('underlines')
+            if len(self.errlines) > 0:
+                # print('underlines')
                 self.view.settings().set("oops", oops_map)
-                self.view.add_regions("oops", self.underlines, "illegal", "", sublime.DRAW_NO_FILL + sublime.DRAW_NO_OUTLINE + sublime.DRAW_SQUIGGLY_UNDERLINE)
-                self.view.window().run_command("show_panel", {"panel": "output.variable_get"})
+                self.view.add_regions("oops", self.errlines, "sublimelinter.mark.error", "circle",  sublime.DRAW_NO_FILL|sublime.DRAW_NO_OUTLINE|sublime.DRAW_SOLID_UNDERLINE )
+                if bool(helpers.get_settings(self.view,'omnisharp_onsave_showerrorwindows')):
+                    self.view.window().run_command("show_panel", {"panel": "output.variable_get"})
+            if len(self.warninglines) > 0:
+                # print('underlines')
+                self.view.settings().set("oops", oops_map)
```

Comment:
I think we should still show warnings if  `bool(helpers.get_settings(self.view,'omnisharp_onsave_showerrorwindows'))` is `true`